app/views/items/index.html.erb

```diff
@@ -22,20 +22,32 @@
                   <th>Description</th>
                   <th>Status</th>
                   <th>Actions</th>
+                  <th>Lender</th>
+                  <th>Borrower</th>
+                  <th>Due Date</th>
                 </tr>
              </thead>
             <% @items.each do |item| %>
-            <tr>
-              <th><%= item.id %></th> 
-              <th><a href="<%= edit_item_path item %>"><%= item.description %></a></th>   
-              <% if item.current %>
-                <th class="checked-out">
-                  <%= link_to "Check-in", checkin_log_path(item.current), data: { confirm: 'Are you sure?' } %></th>
-              <% else %>
-                <th class="available">Available</th>
-              <% end %>
+              <tr>
+                <th><%= item.id %></th> 
+                <th><a href="<%= edit_item_path item %>"><%= item.description %></a></th>   
+                <% if item.current %>
+                  <th class="checked-out">
+                    <%= link_to "Check-in", checkin_log_path(item.current), data: { confirm: 'Are you sure?' } %></th>
+                <% else %>
+                  <th class="available">Available</th>
+                <% end %>
                 <th><%= link_to 'delete', item, method: :delete, data: { confirm: 'Are you sure?' } %></th>
-            </tr>
+                <% if item.current %>
```

Comment:
Im not sure about this bit when you are doing User.find you are querying the db again which is expensive. It would be much nicer if its possible to do some like item.user.name????? How else could you do this?

Reply:
The line on 41 is actually from the original code. Do you still think we should modify it? Thanks

Reply:
Line 42 to 43 is the line I was talking about